src/parser/nscontext.js

```diff
@@ -206,7 +206,7 @@ class NamespaceContext {
         return mapping;
       }
     }
-    if (this.getNamespaceURI(prefix, true)) {
+    if (this.getNamespaceURI(prefix)) {
```

Comment:
@raymondfeng, suppose we have a parent element with prefix ns1 (xmlns: "http://AAA") and a child with prefix ns1 (xmlns: "http://XYZ"), it might be misleading to read the xml document